iso15118/secc/comm_session_handler.py

```diff
@@ -308,11 +308,16 @@ async def end_current_session(self, peer_ip_address: str):
         self.udp_server.resume_udp_server()
 
     async def start_tcp_server(self, with_tls: bool):
-        if self.tcp_server_handler is not None:
-            """A TCP server is already available, ready to respond.
-            (Perhaps created when the last SDP request was received.)
-            """
-            return
+        if self.tcp_server_handler and self.tcp_server.is_tls_enabled == with_tls:
+            return True
```

Comment:
do we need to return it as True, it seems just 'return'

Reply:
yes, fixed.